### DISCUSSION TITLE
How do I change colour of menu/title bar?

### DISCUSSION THREAD

**Discussion by Zineffable1** [brackets-cont/brackets#176] [Q&A]
How do I change the colour of where the red arrow is?
![image](https://user-images.githubusercontent.com/61528972/161386021-0ed34d9f-e7a9-458a-b600-f59a1787b691.png)

**Comment by charlypa** [comment]
Click view->Themes. Select the theme of your choice.

**Reply by Zineffable1** [reply]
There is only light and dark, neither of them change the colour of the specified area.
